src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { css } from "@emotion/css";
+import { useTheme2 } from "@grafana/ui";
+import { AuthMethod, CustomMethod, CustomMethodId } from "./types";
+import { AuthMethodSettings } from "./auth-method/AuthMethodSettings";
+import { TLSSettings, Props as TLSSettingsProps } from "./tls/TLSSettings";
+import { Props as BasicAuthProps } from "./auth-method/BasicAuth";
+import {
+  CustomHeaders,
+  Props as CustomHeadersProps,
+} from "./custom-headers/CustomHeaders";
+
+export type Props = {
+  selectedMethod: AuthMethod | CustomMethodId;
+  mostCommonMethod?: AuthMethod | CustomMethodId;
+  visibleMethods?: (AuthMethod | CustomMethodId)[];
+  customMethods?: CustomMethod[];
+  onAuthMethodSelect: (authType: AuthMethod | CustomMethodId) => void;
+  basicAuth?: BasicAuthProps;
+  TLS?: TLSSettingsProps;
+  customHeaders?: CustomHeadersProps;
+};
+
+export const Auth: React.FC<Props> = ({
+  selectedMethod,
+  mostCommonMethod,
+  visibleMethods: visibleMethodsFromProps,
+  customMethods,
+  onAuthMethodSelect,
+  basicAuth,
+  TLS,
+  customHeaders,
+}) => {
+  const { colors } = useTheme2();
+
+  const visibleMethods: (
+    | AuthMethod
+    | CustomMethodId
+  )[] = visibleMethodsFromProps ?? [
+    AuthMethod.BasicAuth,
+    AuthMethod.CrossSiteCredentials,
```

Comment:
Nit. I think I mentioned this in our last discussion. Not sure this should be default given it's only applicable to `Access mode=Browser`, i.e. making a direct connection to datasource in the browser not going through Grafana backend. In core Grafana most of datasources have removed support for `Access mode=Browser` and now only support `Access mode=Server`. Prometheus is one example.  Seems like the theme moving forward is that we want to remove the support for `Access mode` permanently and by default use `Access mode=Server`.

Reply:
We decided to hide this by default. Thanks, will update the PR.

Reply:
Done in 4f68ada7bd10b7221de62eb8dda6613880028a6a.